.github/workflows/ci.yml

```diff
@@ -41,23 +41,10 @@ jobs:
       - name: Code sniff
         run: composer run-script code:lint
 
-      - name: Install Terminus
```

Comment:
Is terminus already installed at another step?

Reply:
It's in the docker image